File: fluoxetine_backstage/src/main/resources/static/assets/js/service/noteService.js
app.service('noteService',function ($http) {

    this.search = function (entity,page,size) {
      return $http.post('/note/search?page='+page+'&size='+size,entity);
    };

    //修改状态
    this.updateStatus = function (ids,status) {
      return $http.get('/note/updateStatus?ids='+ids+'&status='+status);
    };

    this.updateRecomm = function (ids,status) {
      return $http.get('/note/updateRecomm?ids='+ids+'&status='+status);
    };

    this.findOne = function (id) {
      return $http.get('/note/findOne?id='+id);
    };

    //查找评论
    this.findTalk = function (noteId) {
        return $http.get('/note/findTalk?noteId='+noteId);
    };

    //删除评论
    this.deleteTalk = function (id) {
      return $http.get('/note/deleteTalk?id='+id);
    };
});

File: fluoxetine_backstage/src/main/java/cn/fluoxetine/fluoxetinebackstage/service/NoteService.java
package cn.fluoxetine.fluoxetinebackstage.service;

import cn.fluoxetine.fluoxetinebackstage.pojo.FtbNote;
import cn.fluoxetine.fluoxetinebackstage.pojo.PageResult;

/**
 * @author 11699
 * @date 2020/2/10 - 15:29
 */
public interface NoteService {

    public PageResult search(FtbNote note,int pageNum,int pageSize);

    /**
     * 修改状态
     * @param ids
     * @param status
     */
    public void updateStatus(Integer[] ids,Integer status);


    /**
     * 修改推荐状态
     * @param ids
     * @param recomm
     */
    public void updateRecomm(Integer[] ids,Integer recomm);

    /**
     * 查找
     * @param id
     * @return
     */
    public FtbNote findOne(Integer id);

}
